### ISSUE TITLE
Stuck in Razorpay gateway after successful transaction Android 

### ISSUE THREAD

**Issue by vetriselvi-i2i** [razorpay/razorpay-flutter#176]
Please provide all the information requested. Issues that do not follow this format are likely to stall.

## Description
Making the payment with Card/Phonepe via Razorpay gateway. 
Phonepe - After payment success Razorpay still in the same page (like again I've to make payment screen)
Card payment - Not even processing any cards (Both master & visa cards) OTP screen is not coming.

## RazorPay Version :
com.razorpay:checkout:1.6.6

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. Choose Razorpay -> UPI -> Phonepe -> {pay in phonepe} -> success - automatically coming back to Razorpay from phonepe -> app is still in the same page. 

## Expected Results
Razorpay gateway should close with success callback

**Comment by sumedht** [comment]
@vetriselvi-i2i Does not getting any callback?

**Comment by vetriselvi-i2i** [comment]
> @vetriselvi-i2i Does not getting any callback?

Yes @sumedht Not getting a callback and the Razorpay window not closing. both the cases.

**Comment by hashimchargemod** [comment]
im getting a success page from razorpay but _razorpay.on(Razorpay.EVENT_PAYMENT_SUCCESS, _handlePaymentSuccess); and handle payment success is not popping up.
[connection] nw_resolver_start_query_timer_block_invoke [C5.1] Query fired: did not receive all answers in time for lumberjack-metrics.razorpay.com:443
[connection] nw_resolver_start_query_timer_block_invoke [C4.1] Query fired: did not receive all answers in time for butler.razorpay.com:443
[connection] nw_resolver_start_query_timer_block_invoke [C6.1] Query fired: did not receive all answers in time for cdn.razorpay.com:443
[connection] nw_resolver_start_query_timer_block_invoke [C7.1] Query fired: did not receive all answers in time for lumberjack.razorpay.com:443
[connection] nw_resolver_start_query_timer_block_invoke [C8.1] Query fired: did not receive all answers in time for play.googleapis.com:443
[connection] nw_resolver_start_query_timer_block_invoke [C9.1] Query fired: did not receive all answers in time for play.googleapis.com:443
/Users/runner/work/razorpay-pod/razorpay-pod/RazorpayIOS/CheckoutOtpelf/Classes/RazorpayCheckoutVC.swift deinitialized
this is what is showing in debug console
